contract-ui/tabs/overview/components/PermissionsTable.tsx

```diff
@@ -93,14 +93,16 @@ export const PermissionsTable: React.FC<PermissionsTableProps> = ({
               <Center py={4}>
                 <Flex align="center" gap={2}>
                   <Text size="body.md" fontStyle="italic">
-                    {"no permissions to show"}
+                    {allRoleMembers.isLoading
+                      ? "loading permissions"
```

Comment:
Let's use proper capitalization for both here "Loading permissions..." and "No permissions found" (found sounds better to me than "to show")